shard.yml

```diff
@@ -7,11 +7,11 @@ authors:
 development_dependencies:
   spectator:
     gitlab: arctic-fox/spectator
-    version: ~> 0.9.23
+    version: 0.9.38
```

Comment:
```suggestion
    version: ">= 0.9.38"
```

Reply:
Do we really need to use latest version of shards in projects? In such case there is a risk to face similar issue in the future again. And it wasn't obvious that problem in some specific shard. I was lucky that we have only one dependency, but what if a project has 10-20 and any of them can be updated suddenly. I prefer to update deps manually when it is really needed. And make sure that specs are passed, nothing broken after that and only after that lock the version.

+ @Adem-Niagbe

Reply:
I'm pretty sure you don't have any serious process/es re: systematic update of the deps so these will just become stale after time. Using `~>` operator gives us enough safety from the API breakage and bugs like the one you've encountered are occasional and could happen anytime. This is the first case like this and I don't see why should it necessitate locking. Your arguments are contrary to my experiences with many shards already, so all in all I'm against pinning altogether (unless necessary).

Reply:
@Sija how confident are you with the proposed idea, as there is a potential risk that this will not work, basing on the experience.
If the risk is minimal - we can accept it, but @bararchy this PR should be approved to move further.

Reply:
Well, there's no guarantee, but as the historical evidence shows this risk is minimal and easily mitigated - in case.

Reply:
I have another experience, which wasn't so positive regards 3rd party components, that's why I prefer to update deps having the reason to do that. If project is not actively developed like this one, I also don't see reasons update deps just because. It is better to monitor what's new released in one or another dependency and if it is really useful for project, update the dep.

Reply:
I don't know about your previous experiences but this shard has literally 2 dependencies, and it's a first case of such issue from the beginning of this shard's existence, so in case incompatibilities like these emerges again this would be a quick fix - like you did above.

Reply:
Also, having this issue helped to highlight a bug in the Spectator shard - which's worthy to note in terms of OSS contributions.

Reply:
that's a good point, agree about that

Reply:
Then lat's make it happen. So approval of this PR is required.